modules/persistence/src/services/metadata/ToC/utils/prefixFromEnvironment.ts

```diff
@@ -0,0 +1,9 @@
+import { BranchEntry } from '../../associated_products';
+
+export const prefixFromEnvironment = (repoBranchEntry: BranchEntry) => {
```

Comment:
`BranchEntry` does not have specified keys `url` and `prefix`

also passing in a repoBranchEntry from db, which also does not have the two attributes.

referring to this comment, we should separate logic for pulling url and prefix info, apart from {project-name: {branch-name: repoBranch}}

Reply:
Hmmm, I've made the mistake of conflating the `repos_branches.branches` object vs. the parent `repos_branches` entry - I'll need to adjust the typing here - I'll think about if we need to generalize pulling info for the `url` and `prefix`, I'm mostly convinced but need to reason about when/where that responsibility should be

Reply:
sorry, linked the previous comment (forgot the link before)